feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/jsf-injection/main/module.xml

```diff
@@ -34,7 +34,7 @@
 
     <dependencies>
         <module name="com.sun.jsf-impl"/>
-        <module name="javax.api"/>
+        <module name="java.naming"/>
         <module name="org.jboss.as.ee"/>
```

Comment:
``wildfly-jsf-injection`` seems to need ``java.desktop`` for the ``java.beans`` package.